hera_qm/xrfi.py

```diff
@@ -10,6 +10,7 @@
 from .uvflag import UVFlag
 from . import utils as qm_utils
 from .version import hera_qm_version_str
+from .utils import process_ex_ants
```

Comment:
do we need this import? is if possible to just call `qm_utils.process_ex_ants` since you already have utils imported? I guess it doesn't really matter either way to be honest.

Reply:
good catch. will change.

Reply:
Actually, I like Paul's suggestion to put process_ex_ants in metrics_io, so that will supersede this.

Reply:
:+1: